Automattic/wp-calypso
# Fixed the setup page

Related to # dsp 1164

## Proposed Changes
we are changing the design of the blaze setup page (when there is no connection to the jetpack services) 

the new design looks like the below (screenshots from figma)

![Screenshot 2024-05-02 at 14 39 16](https://github.com/Automattic/wp-calypso/assets/1416426/981b8446-0466-4f04-bcc1-a5f8c6b184ee)

![Screenshot 2024-05-02 at 14 39 09](https://github.com/Automattic/wp-calypso/assets/1416426/e21a19b5-aeca-48aa-ba75-7761859566fd)

![Screenshot 2024-05-02 at 14 38 51](https://github.com/Automattic/wp-calypso/assets/1416426/e5e17d90-9e10-40bc-bd11-39f2bbf5e7c3)

the below are screenshots from the browser with the changes from this PR
![Screenshot 2024-05-03 at 16 18 09](https://github.com/Automattic/wp-calypso/assets/1416426/0c378151-1c7e-4bbe-a85f-daae1deddc65)
![Screenshot 2024-05-03 at 16 13 35](https://github.com/Automattic/wp-calypso/assets/1416426/9cf1eaa2-ec70-4fe2-bc57-24bf94562f01)
![Screenshot 2024-05-06 at 14 33 09](https://github.com/Automattic/wp-calypso/assets/1416426/8b5ff6c6-16b3-4321-85d2-a0c6d097a069)

please note that the texts have been improved a bit (so are not the same as the figma files - check the convo below)


## Testing Instructions

I think the easier way to test this is to open up the woo-blaze plugin locally, and sync the blaze-dashboard to the sandbox (hosts must be enabled for widgets.wp.com to target the sandbox too)

change the return value in includes/class-blaze-dashboard.php line 107 to be either one of the below:

blaze_disabled
site_private_or_coming_soon
disconnected

refresh the blaze dashboard page (in marketing -> blaze) and check the result
## Pre-merge Checklist
